client/src/app/site/pages/meetings/pages/motions/pages/workflows/components/workflow-detail-sort/workflow-detail-sort.component.html

```diff
@@ -0,0 +1,18 @@
+<os-detail-view [collection]="COLLECTION" (idFound)="onIdFound($event)">
+    <os-head-bar [nav]="false" [editMode]="hasChanges" (saveEvent)="save()" (mainEvent)="onCancel()">
+        <div class="title-slot">
+            <h2>{{ 'Sort workflow states' | translate }}</h2>
+        </div>
+    </os-head-bar>
+
+    <mat-card class="os-card">
+        <os-sorting-list [input]="workflowStatesObservable" (sortEvent)="onSorting($event)" #sorter>
+            <ng-template let-state let-index="index">
+                <div class="action-title">
+                    <div>{{ state.name | translate }}</div>
+                    <div *ngIf="index === 0">{{ 'First state' | translate }}</div>
```

Comment:
Since the first state is marked explicitely, we should also update it one save here and therefore fix https://github.com/OpenSlides/openslides-client/issues/1403 directly in this PR